sycl/include/sycl/context.hpp

```diff
@@ -177,13 +177,18 @@ class __SYCL_EXPORT context : public detail::OwnerLessBase<context> {
   ///
   /// The return type depends on information being queried.
   template <typename Param
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
 #if defined(_GLIBCXX_USE_CXX11_ABI) && _GLIBCXX_USE_CXX11_ABI == 0
             ,
             int = detail::emit_get_backend_info_error<context, Param>()
+#endif
 #endif
             >
+  __SYCL_DEPRECATED(
+      "All current implementations of get_backend_info() are to be removed. "
+      "Use respective variants of get_info() instead.")
```

Comment:
```suggestion
#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
  __SYCL_DEPRECATED(
      "All current implementations of get_backend_info() are to be removed. "
      "Use respective variants of get_info() instead.")
#endif
```
After preview changes have been elevated, the deprecated queries are no longer available, so we should free it up again. `get_backend_info` is not deprecated.

**This applies to all the other cases where `__SYCL_DEPRECATED` was added.**

Reply:
Fixed, thanks for pointing this out!